client/my-sites/checkout/checkout-thank-you/licensing-thank-you-auto-activation.tsx

```diff
@@ -0,0 +1,236 @@
+	);
+
+	useEffect( () => {
+		if ( supportTicketStatus === 'success' ) {
+			const thankYouCompletedUrl = addQueryArgs(
+				{
+					siteId: jetpackTemporarySiteId,
+					receiptId,
+				},
+				`/checkout/jetpack/thank-you-completed/no-site/${ productSlug }`
+			);
+			page( thankYouCompletedUrl );
+		} else if ( supportTicketStatus === 'failed' ) {
+			setError(
+				translate( 'There was a problem submitting your website address, please try again.' )
+			);
+		}
+	}, [ jetpackTemporarySiteId, receiptId, supportTicketStatus, productSlug, translate ] );
+
+	const siteSelectOptions =
+		jetpackSites &&
+		jetpackSites.map( ( site: JetpackSite ) => ( {
+			value: site?.URL,
+			label: site.URL,
+			props: {
+				key: site.ID,
+				selected: site.URL === selectedSite,
+				value: site.URL,
+				onClick: () => {
+					setSelectedSite( site.URL );
+				},
+			},
+		} ) );
+
+	const lastSelectOption = {
+		value: 'activate-license-manually',
+		label: translate( "I don't see my site. Let me configure it manually" ),
+		props: {
+			key: 'activate-license-manually',
+			selected: 'activate-license-manually' === selectedSite,
+			value: 'activate-license-manually',
+			onClick: () => {
+				setSelectedSite( 'activate-license-manually' );
+			},
+		},
+	};
+	const selectDropdownItems = siteSelectOptions && [
+		...siteSelectOptions,
+		...[ lastSelectOption ],
+	];
+	const selectedItem = find( selectDropdownItems, 'props.selected' );
+
+	return (
+		<Main fullWidthLayout className="licensing-thank-you-auto-activation">
+			<PageViewTracker
+				options={ { useJetpackGoogleAnalytics: true } }
+				path={ '/checkout/jetpack/thank-you/licensing/:product' }
+				properties={ { product_slug: productSlug } }
+				title="Checkout > Jetpack Thank You Licensing Auto Activation"
+			/>
+			<Card className="licensing-thank-you-auto-activation__card">
+				<div className="licensing-thank-you-auto-activation__card-main">
+					<JetpackLogo size={ 45 } />
+					{ hasProductInfo && <QueryProducts type="jetpack" /> }
+					<h1 className="licensing-thank-you-auto-activation__main-message">
+						{ translate( 'Thank you for your purchase!' ) }{ ' ' }
+						{ String.fromCodePoint( 0x1f389 ) /* Celebration emoji 🎉 */ }
+					</h1>
+					{ hasProductInfo && ( isProductListFetching || productName ) && (
+						<p
+							className={
+								isProductListFetching
+									? 'licensing-thank-you-auto-activation__product-info-loading'
+									: 'licensing-thank-you-auto-activation__product-info'
+							}
+						>
+							{ translate( 'Hello %(username)s,', {
+								args: {
+									username: userName,
+								},
+							} ) }
+							<br />
+							{ translate( 'Select the site you want %(productName)s on:', {
+								args: {
+									productName,
+								},
+							} ) }
+						</p>
+					) }
+					<SelectDropdown
+						className="licensing-thank-you-auto-activation__select"
+						selectedText={ selectedItem ? selectedItem.label : translate( 'Select…' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **125** times:
`translate( '- Select -' )` *ES Score: 6*